week2/homework/src/actions/addTodo.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const chalk = require('chalk');
+const helpMessages = require('./helpMessages');
+const storeTodos = require('./storeTodos');
+
+const args = process.argv.slice(2);
```

Comment:
You should have passed arguments that each action requires instead of reading a global object.